packages/json_utils.ol

```diff
@@ -29,6 +29,20 @@ type GetJsonValueRequest: any {
 }
 type GetJsonValueResponse: undefined
 
+type ValidateJsonRequest: void {
+	json: string 
+	schema: string 
+}
+
+type ValidateJsonResponse: void {
```

Comment:
You don't need `: void` (and similarly below).

In general I'd avoid writing `: void` any longer for void nodes that contain subnodes.

Reply:
Done